docs/eportal/README.md

```diff
@@ -1046,7 +915,7 @@ forward requests to ePortal instance. The only requirement is to provide correct
 
 For example balancer nginx config:
 
-```
+```json
```

Comment:
It's nginx config not JSON. It's better to rely on auto code syntax feature of the markdown renderer and not specify lang tag at all.